pkg/labels/labels.go

```diff
@@ -51,6 +51,16 @@ const (
 	// with IDNameHost if the kube-apiserver is running on the local host.
 	IDNameKubeAPIServer = "kube-apiserver"
 
+	// IDNameEncryptedOverlay is the label used to identify encrypted overlay
+	// traffic.
+	//
+	// It is part of the reserved identity 11 and signals that overlay traffic
+	// with this identity must be IPSec encrypted before leaving the host.
+	//
+	// This identity should never be seen on the wire and is used only on the
+	// local host.
+	IDNameEncryptedOverlay = "encrypted-overlay"
```

Comment:
Is that the name that will show up in `cilium monitor`? Should it be `overlay-to-encrypt`?

Reply:
So far all the naming has been "encrypted overlay". Why the change now to 'overlay-to-encrypt' ?

Reply:
I'm just a bit worried it will look like the traffic is encrypted when looked at with `cilium monitor`. I wasn't suggesting to change the name everywhere else. And maybe it's fine because only devs use `cilium monitor` nowadays?

Reply:
Ah I see, if its just for Cilium monitor output then I think that's cool. I'll make the change.

Reply:
Fixed in latest push